osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -2874,6 +2874,32 @@
         "source_denom": "ubld"
       }
     },
+    {
+      "description": "IST is a decentralized and fully collateralized stable token built for the Cosmos ecosystem.",
+      "denom_units": [
+        {
+          "denom": "ibc/5973C068568365FFF40DEDCF1A1CB7582B6116B731CD31A12231AE25E20B871F",
+          "exponent": 0,
+          "aliases": ["uist"]
+        },
+        {
+          "denom": "ist",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/5973C068568365FFF40DEDCF1A1CB7582B6116B731CD31A12231AE25E20B871F",
```

Comment:
I think this needs changing. I believe SHA256(transfer/channel-320/uist) = 
92BE0717F4678905E53F4E45B2DED18BC0CB97BF1F8B6A25AFEDF3D5A879B4D5

Reply:
I grabbed the wrong asset from my wallet. Thanks! Fixed